routes/web.php

```diff
@@ -403,9 +385,15 @@ function () : void {
         'middleware' => ['auth', 'role:admin']
     ],
     function (): void {
-        Route::get('jobs/create/as-manager/{manager}', 'JobController@createAsManager')
-            ->middleware('can:create,App\Models\JobPoster')
-            ->name('admin.jobs.create.as_manager');
+        /* Edit Job */
```

Comment:
Do these not need to be localized? I know right now it's probably only one or two people using the admin portal for edits, but we have the translations handy. It also seems to be affecting other routing with the menus and such (logout triggers a 405 from these pages currently).

Reply:
You're right, it makes more sense to localize this page.
I'll also fix the menu, so it acts like its on the admin portal, not the applicant portal.